docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -1415,6 +1415,11 @@ The following objects can be passed to recorders:
 - Any arbitrary object via the `io.quarkus.deployment.recording.RecorderContext#registerSubstitution(Class, Class, Class)` mechanism
 - Arrays, Lists and Maps of the above
 
+==== Injecting Configuration into Recorders
+
+Configuration objects with phase `RUNTIME` or `BUILD_AND_RUNTIME_FIXED` can be injected into recorders via constructor
+injection. Just create a constructor that takes the configuraiton objects the recorder needs. If the recorder has multiple
```

Comment:
So, is there any reason why we limit this to constructor injection and not field injection?

Reply:
Because I like final fields, and it is easier to implement. We could add it if there is demand.